File: tutorial/java/stdlib/io/Io.java
// SPDX-License-Identifier: BSD-3-Clause

import java.io.File;
import java.io.IOException;
import java.io.PrintWriter;
import java.util.Scanner;

public class Io {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(new File("in"))) {
            int x = sc.nextInt();
            int a = sc.nextInt();
            int b = sc.nextInt();
            int n = sc.nextInt();
            String c = sc.next();
            String s = sc.next();

            try (PrintWriter pw = new PrintWriter(new File("out"))) {
                pw.println(x + " " + a + " " + b + " " + c + " " + s);
            }
            System.out.println("wrote to out");
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }
}

File: tutorial/python/stdlib/io/io.py
#!/usr/bin/env python3
# SPDX-License-Identifier: BSD-3-Clause


def main() -> None:
    with open("in") as fin:
        x: int = int(fin.readline().strip())
        tokens: list[str] = fin.readline().split()
        a: int = int(tokens[0])
        b: int = int(tokens[1])
        line: list[str] = fin.readline().split()
        n: int = int(line[0])
        c: str = line[1][0]
        s: str = line[2]

    with open("out", "w") as fout:
        fout.write(f"{x} {a} {b} {c} {s}\n")
    print("wrote to out")


if __name__ == "__main__":
    main()
